src/md/enc/stgio.cpp

```diff
@@ -869,7 +869,7 @@ HRESULT StgIO::MapFileToMem(            // Return code.
 #endif
             
             if ((m_hMapping = WszCreateFileMapping(m_hFile, pAttributes, dwProtectionFlags,
-                0, 0, *m_rcShared ? m_rcShared : 0)) == 0)
+                0, 0, nullptr)) == 0)
```

Comment:
It looks like m_rcShared is never assigned to non-empty string - could you please delete it from everywhere?